docs/api/web-frame-main.md

```diff
@@ -93,6 +93,17 @@ In the browser window some HTML APIs like `requestFullScreen` can only be
 invoked by a gesture from the user. Setting `userGesture` to `true` will remove
 this limitation.
 
+#### `frame.executeJavaScriptInIsolatedWorld(worldId, code[, userGesture])`
+
+* `worldId` Integer - The ID of the world to run the javascript in, `0` is the default world, `999` is the world used by Electron's `contextIsolation` feature.  You can provide any integer here.
```

Comment:
JS numbers can be _huge_ but world_id is an int32_t internally. Kind of a pedantic point, but we should probably use some text other than "any integer" here.